docker/build_support/build_image

```diff
@@ -0,0 +1,118 @@
+#! /usr/bin/env bash
```

Comment:
HOKAY.  The new build_image is a lot simpler, half the number of lines, and does involve docker inside of docker non sense.  Check the above deleted `build_image` logic for comparison.